ming_sim/action_materialize.py

```
@@ -883,6 +884,11 @@ def stage_punishment_candidate(
         "punish_action": action,
         "mode": mode,
     }
+    # #658：与 durable apply 共吃 require_backing_dossier_id，禁第二份 int/存在性分支
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)